### PR TITLE
Mobile menu navigation

### PR DESCRIPTION
**Pull request by sangwa7**
### Mobile menu navigation

-Add mobile menu list
-Add mobile menu interaction with JavaScript functions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,23 +33,36 @@
...
         <div class="menu">
-          <img
-            src="https://img.icons8.com/color/48/000000/menu--v5.png"
-            alt=""
-          />
+          <div>
+            <a href="javascript:void(0);" id = "menu-button">
+            <img src="https://img.icons8.com/color/48/000000/menu--v5.png"
+            alt="Menu icon for mobile view" onclick="mobileMenu()"
+          />
```
Comment (ShoiraTa) [inline index.html:57]: > You have done an amazing job in this project setting up both, JS and CSS. But there are some issues related to the project requirements I will list them below. 

- [ ] Please make sure that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is be displayed. Currently there are no links inside `li` tags. 
- [ ] Please make sure that when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- JS is working as expected! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:57]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Well done implementing all the changes requested by your previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**